src/Acuminator/Acuminator.Tests/Verification/DiagnosticResultExtensions.cs

```diff
@@ -15,26 +15,47 @@ public static class DiagnosticResultExtensions
 			return CreateDiagnosticResult(descriptor, messageArgs, (line, column));
 		}
 
-		public static DiagnosticResult CreateFor(this DiagnosticDescriptor descriptor, 
-			(int line, int column) location,
-			(int line, int column) extraLocation,
+		public static DiagnosticResult CreateFor(this DiagnosticDescriptor descriptor,
+			(int Line, int Column) location,
+			(int Line, int Column) extraLocation,
 			params object[] messageArgs)
 		{
 			return CreateDiagnosticResult(descriptor, messageArgs, location, extraLocation);
 		}
 
+		public static DiagnosticResult CreateFor(this DiagnosticDescriptor descriptor,
+			IEnumerable<(int Line, int Column)> locations)
+		{
+			return CreateDiagnosticResult(descriptor, null, locations);
+		}
+
+		private static DiagnosticResult CreateDiagnosticResult(DiagnosticDescriptor descriptor,
+			object[] messageArgs,
+			params (int Line, int Column)[] locations)
```

Comment:
Split this argument to the main location and the list of the extra locations (same for the second method below)